RetailCoder.VBE/Inspections/UntypedFunctionUsageInspection.cs

```diff
@@ -43,10 +43,13 @@ public UntypedFunctionUsageInspection(RubberduckParserState state)
         public override IEnumerable<InspectionResultBase> GetInspectionResults()
         {
             var declarations = BuiltInDeclarations
-                    // note: these *should* be functions, but somehow they're not defined as such
-                .Where(item => _tokens.Any(token => (item.IdentifierName == token || item.IdentifierName == "_B_var_" + token)) && item.References.Any());
+                // note: these *should* be functions, but somehow they're not defined as such
+                .Where(item =>
+                        _tokens.Any(token => item.IdentifierName == token || item.IdentifierName == "_B_var_" + token) &&
+                        item.References.Any(reference => _tokens.Contains(reference.IdentifierName)));
```

Comment:
I'll take it, but doesn't this merely *work around* the fact that stale `IdentifierReference` instances are in the parser state?